app/routes/api.mcp-update-config.ts

```
@@ -0,0 +1,170 @@
+
+    if (!action) {
+      return json({ error: 'Action is required' }, { status: 400 });
+    }
+
+    switch (action) {
+      case 'add_server':
+        if (!serverData) {
+          return json({ error: 'Server data is required' }, { status: 400 });
+        }
+
+        const { name, description, endpoint, capabilities, config } = serverData;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)